### PR TITLE
Another version for Dutch media sources

### PR DESCRIPTION
**Pull request by sh4wn**
Apparently I'm already the third. Forgot to check if there were other pull requests for the Netherlands. I think this one is a bit more complete than the others though. 

Suggestions welcome.

### CODE REVIEW

**Comment by vecna** [comment]
Thank you, I'll try to understand which media list is better/more complete #38 #39 #40

File: verified_media/thenetherlands [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+http://dvhn.nl
+http://ed.nl/
+http://destentor.nl
+http://omroepwest.nl
+http://omroepgelderland.nl
+http://omroepbrabant.nl
+http://omropfryslan.nl
+http://rtvnh.nl
+http://omroepzeeland.nl
+http//omroepflevoland.nl
```
Comment (lhoBas) [inline verified_media/thenetherlands]: typo: http://

File: verified_media/thenetherlands [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+http://nu.nl
+http://rtlnieuws.nl
+http://nos.nl
+http://tubantia.nl/
+http://bnr.nl
+http://blikopnieuws.nl
+http://nieuws.nl
+http://dutchnews.nl/
+http://tweakers.net
+http://blende.com
```
Comment (lhoBas) [inline verified_media/thenetherlands]: typo: http://blendle.com/

**Comment by sh4wn** [comment]
Thanks, I've fixed them!

**Comment by lhoBas** [comment]
Don't forget to push your commit with the fixes ;)

**Comment by sh4wn** [comment]
They're pushed, but I guess the pull request doesn't update anymore now that it's closed ;)

**Comment by vecna** [comment]
yes yes, I've closed by hand the three PR because they will have conflicted, and because there were some duplicated website with or without "www." on top.

I've received some results from someone of you, but was a partial list, now I'm collecting from greenhost provider. If someone has an home access, please run the script so we can compare different AS :)

**Comment by sh4wn** [comment]
Yeah, I've ran it on my VPS, and on my home computer (but at first without traceroute installed), so that's why probably one is incomplete.

**Pull request closed without merging**